src/views/Visualizer/View/search.js

```diff
@@ -0,0 +1,44 @@
+const fetchNominatim = async (query, language = 'en', t, searchProvider, viewbox = []) => {
```

Comment:
When creation function with many arguments, it may be preferable to use an object for passing them instead.

```suggestion
const fetchNominatim = async ({
  query,
  t,
  searchProvider,
  language = 'en',
  viewbox = [],
}) => {
```